history/cultures/pandaren_group.txt

```diff
@@ -62,4 +63,6 @@
 	#
 	discover_innovation = innovation_windmills
 	discover_innovation = innovation_currency_03
+	#
+	discover_innovation = innovation_cloud_serpents
```

Comment:
Pandarens should have serpents from the beginning. Their taming happened around 11,900 ago.
https://wowpedia.fandom.com/wiki/Cloud_serpent#Background